src/pykeen/evaluation/evaluator.py

```diff
@@ -267,11 +269,14 @@ def _param_size_search(
             values_dict[key] = start_value
             values_dict['slice_size'] = None
         elif key == 'slice_size':
+            if start_value is None:
+                raise ValueError('must specify a start value with key="slice_size"')  # TODO @lvermue check
```

Comment:
@lvermue please check that this is correct. Can the start value be none if the key is slice_size? If so, what happens?

Reply:
@cthoyt When the key slice_size is set to `None` it simply won't slice. This leads to the code first trying `batch_size=1` with no slicing again and afterwards probably failing while trying to integer divide `None` by 2. Currently the start_value is always set in https://github.com/pykeen/pykeen/blob/621341568b90c98dadb94c912614e0d0ceac0b18/src/pykeen/evaluation/evaluator.py#L236
and that's the most reasonable valuable one can possibly set it to.

Reply:
so this code will never raise the error, or should we change what's going on?

Reply:
Right now the master should never raise the error. But it's not foolproof, because it expects that the calling function gives the right parameter to start with.

Reply:
Okay since we have mypy to check that this is "technically" correct then I'm confident enough as long as all of the tests pass